README.md

```diff
@@ -19,217 +19,170 @@
+
+```bash
+gadgetctl --remote-address=tcp://$IP:1234
+gadgetctl run trace_open
+```
+
+***The above demonstrates the simplest command. To learn how to filter, export, etc. please go consult the documentation for the [run](/docs/guides/run.md) command***.
+ 
+## Core concepts
+
+TODO: Insert system diagram
+
+### What is a Gadget?
+
+Gadgets are the central component in the Inspektor Gadget framework. A Gadget is an [OCI image](https://opencontainers.org/) that includes one or more eBPF programs, metadata YAML file and, optionally, WASM modules for post processing and a logo file.
+As OCI images, they use the same tooling as containers and share the same attributes; shareable, modular, deployable, etc.
+Gadgets are built using the [`ig image build`](/docs/gadg) command.
+
+You can find a growing collection of Gadgets on [Artifact HUB](https://artifacthub.io/packages/search?kind=22). This includes both in-tree Gadgets (hosted in this git repository in the [/gadgets](/gadgets/README.md) directory and third-party Gadgets).
+
+See the [Gadget documentation](/gadgets/README.md) for more information.
 
-## How to use
+#### :warning: For versions prior to v0.31.0
 
-`kubectl gadget --help` will provide you the list of supported commands and their flags.
+Prior to v0.31.0, Inspektor Gadget shipped gadgets in the binary. As of v0.31.0 these ***built-in*** Gadgets are still available and work as before but their use is discourage as they will be deprecate at some point. We encourage users to use ***image-based*** Gadgets going forward as they provide more features and decouple the eBPF programs from the Inspektor Gadget release process.
+
+### What is enrichment?
+
+The data that eBPF collects from the kernel includes no knowledge about Kubernetes, container
+runtimes or any other high-level user-space concepts. In order to relate this data to these high-level
+concepts and make the eBPF data immediately more understandable, Inspektor Gadget automatically
+uses kernel primitives such as mount namespaces, pids, etc. to infer which high-level
+concepts they relate to; Kubernetes pods, container names, DNS names, etc. The process of augmenting
+the eBPF data with these high-level concepts is called enrichment.
+
+Enrichment flows the other way, too. Inspektor Gadget enables users to do high-performance
+in-kernel filtering by only referencing high-level concepts such as Kubernetes pods, container
+names, etc.; automatically translating these to the corresponding low-level kernel resources.
+
+See the [enrichment documentation](docs/enrichment)
+
+### What is an operator
+
+In Inspektor Gadget, an operator is any part of the framework where an action is taken. Some operators are under-the-hood (i.e. fetching and loading Gadgets) while others are user-exposed (enrichment, filtering, export, etc.) and can be reordered and overridden.
+
+See the [operator documentation](docs/operators.md) for more information.
```

Comment:
What do you think about this description instead https://github.com/inspektor-gadget/inspektor-gadget/blob/ece7cf9ab00bef96fed8e93caabe3e3c717da416/docs/core-concepts/_index.md#operators? 

/cc @flyth